src/org/omegat/core/segmentation/SRX.java

```diff
@@ -157,32 +217,65 @@ public static SRX loadSRX(File configFile) {
                     sb.append("\n");
                 }
                 Log.logErrorRB("CORE_SRX_EXC_LOADING_SEG_RULES", sb.toString());
-                res = new SRX();
-                res.initDefaults();
-                return res;
+                return SRX.getDefault();
```

Comment:
The result of `SRX.getDefault()` and `res = new SRX(); res.initDefaults()` is not exactly the same: the former sets `includeEndingTags` and `segmentSubflows` to true, while the latter does not. Is it safe to replace the former with the latter in each situation?

Reply:
Actually you don't implement support for segment subflows nor includeEndingTags, so it has no consequence. But if one day you really implement segment subflows, you will have to ask yourself why the default constructor does not set default values as well.

Reply:
It appears you're right; the members are not actually used for anything substantial. Your change is fine, then.